### ISSUE TITLE
When we navigate using TAB key inside the "Available Authorization" dialog keyboard focus moves to background content after the 'Close' button: A11y_VS Code extension for API Center_Open API Documentation_Available authorizations_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#108]
### GitHub Tags
#A11yTCS; #A11ySev2; #Mac; #WCAG2.4.3; #GH_VSCodeextensionforAPICenter_Mac_Apr2024; #DesktopApp; #VS Code Extension for API Center; #keyboard #A11yMAS; #device; #FocusOrder;#CEACCMASchange;#External;#Thirdparty;#eDAD:3P:Bug222928;#External:SwaggerUI;

### Environment Details:
Application Name: VS Code Extension for API Center
macos sonoma version:14.4.1

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. Tab till azure tool manual testing and click on it.
6. Tab till environment and click on it.
7. Tab till open API and press enter.
6. Go to "Open API Documentation" using the down arrow and press enter.
7. Demo conference API screen will appear.
8. Tab till Authorize button and press enter.
9. Verify whether the When we navigate using TAB key inside the "Available Authorization" dialog keyboard focus moves to background content after the 'Close' button or not.

### Actual Result:
When we navigate using TAB key inside the "Available Authorization" dialog keyboard focus moves to background content after the 'Close' button

### Expected Result:
When we navigate using TAB key inside the "Available Authorization" dialog keyboard focus should not move to background content after the 'Close' button. Also, with "ESC" key we should be able to close the dialog.

### User Impact:
Keyboard users will face difficulty here as they will not be able to understand where the focus is and will not be able to interact with the required controls.

### Attachment:
![Bug_108_gif](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/cec0478c-2cfe-40b6-bbc1-9e136d04ec98)

**Comment by AnKushSingh05** [comment]
@formulahendry Can you please provide us the standalone sample app or Code pen link to route this bug to external team.

**Comment by AnKushSingh05** [comment]
IRT Bug logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222928

**Comment by Yash14j** [comment]
### GitHub Tags:
#External;#External:SwaggerUI;#eDAD:3p:222928;ThirdParty;

**Comment by AnKushSingh05** [comment]
Issue is not fixed in the latest version

**Comment by AnKushSingh05** [comment]
External bug: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222928

**Comment by AnKushSingh05** [comment]
External bug : https://github.com/swagger-api/swagger-ui/issues/10321

**Comment by AnKushSingh05** [comment]
Adding the tag #ResolvedByTT;
